content/docs/0.8.x/operate/upgrade/index.md

```diff
@@ -8,7 +8,7 @@ aliases:
 
 ## Upgrade from Keptn 0.7 to 0.8
 
-Please follow the three steps to succesfully upgrade your Keptn 0.7.3 to 0.8.0 
+Please follow the three steps to succesfully upgrade your Keptn 0.8.0 to 0.8.0 
```

Comment:
Too much replaced :) Please revert this change

Reply:
find-replace issue ;) fixed it.